WORKSPACE

```diff
@@ -15,15 +15,39 @@
 
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
+"""
+Google Cloud C++ API dependencies.
+"""
+
+# Import Google APIs with C++ rules.
 git_repository(
-    name = "gtest",
```

Comment:
Presumably you'll need `gtest` and `abseil-cpp` soon, and so why remove them?

Reply:
These are imported through the com_github_googleapis_... deps already.

https://github.com/googleinterns/cloud-kms-oss-tools/blob/5f7d08435890a98a5a705103a93b12427f503e62/WORKSPACE#L30-L35

Reply:
Fair enough. It might be worth adding a comment explaining this.